### PR TITLE
Update Bottom App Bar Theme Section README.md

### PR DESCRIPTION
**Pull request by mohammadcazig**
Bottom App Bar currently does not support color or typography theming, only FAB and background color is exposed.

### CODE REVIEW

**Review by romoore — CHANGES_REQUESTED** [review]
Can you please add a section about how to theme?

**Comment by mohammadcazig** [comment]
@jverkoey Can you double check if this description is enough? Also should I mark the current color themer as deprecated ? since it doesn't really match our new scheme mapping.

**Comment by jverkoey** [comment]
I added a note to the themer with a link to the pivotal story tracking the work to update the themer. You may want to link to the story here as well.

File: components/BottomAppBar/README.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ import MaterialComponents.MaterialBottomAppBar
 ```
 <!--</div>-->
 
+### How to theme
```
Comment (jverkoey) [inline components/BottomAppBar/README.md]:
```
### Theming
```

File: components/BottomAppBar/README.md [hunk 2]
```diff
@@ -73,6 +73,10 @@ import MaterialComponents.MaterialBottomAppBar
...
+
+Currently BottomAppBar does not provide sufficient API to theme it properly. Supporting API will be provided soon.
```
Comment (jverkoey) [inline components/BottomAppBar/README.md]:
```
BottomAppBar does not support Material Theming yet. Please follow this [tracking issue](https://www.pivotaltracker.com/story/show/157095394) to track progress.
```

**Comment by jverkoey** [comment]
This was automated by https://github.com/material-components/material-components-ios/pull/3562.

**Pull request closed without merging**